debian/rules

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/make -f
+
+DISTRIBUTION = $(shell sed -n "s/^VERSION_CODENAME=//p" /etc/os-release)
+VERSION = 11.0.0-1
+PACKAGEVERSION = $(VERSION)-0~$(DISTRIBUTION)0
+TARBALL = wait4it-$(VERSION).tar.gz
+URL = https://github.com/ph4r5h4d/wait4it/
+
+%:
+		dh $@
+override_dh_auto_clean:
+override_dh_auto_test:
+override_dh_auto_build:
+override_dh_auto_install:
+	go mod download
+	go build -o bin/wait4it
```

Comment:
its better to use the same method for all builds. my suggestion is to use `go run main.go build` here as well.

Reply:
I'll update it, this is just a draft that works with the old version.